scripts/backup/run_backup/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# python2 requirements
+
+psycopg2
+boto3
+paramiko
```

Comment:
This is a great start. Can we please specify the versions that we tested with?

Reply:
Good idea. Done.